py/desimodel/test/test_healpix.py

```diff
@@ -18,10 +18,10 @@ def test_tiles2pix(self):
         tiles = desimodel.io.load_tiles()
 
         pix = tiles2pix(nside=8, tiles=tiles[:3], radius=1.6)
-        self.assertTrue( np.all( (pix, [196,197,208,302,303]) ) )
```

Comment:
odd; your original syntax is what I thought was correct and passes without deprecation warnings with numpy 1.11.2 on my laptop.  Perhaps numpy 1.12 on Travis is warning about an upcoming change to how equality between an array and a list will be evaluated.

The second syntax also passes on my laptop, but that is accidental: pix and the array individually evaluate to True and pass np.all, even if they are different.  I don't understand why those failed on Travis.

I've added debugging printouts and tried wrapping the list in np.array() ...